.github/workflows/publish.yaml

```diff
@@ -0,0 +1,14 @@
+# A CI configuration to auto-publish pub packages.
+
+name: Publish
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    tags: [ 'v[0-9]+.[0-9]+.[0-9]+' ]
```

Comment:
Should we change our existing tags?

I suspect we created them without the `v` prefix, because of cargo-culting.
Regardless, of the fact that docs says to prefer the `v` prefix for newer repositories.

Reply:
Github (somewhere) says they prefer this, too.

Reply:
Don't change existing tags. Not good git-fu. We'll just "do the right thing" moving forward.